### PR TITLE
Rename mbedmicro-rtos-mbed to mbed_rtos

### PR DESCRIPTION
**Pull request by rajkan01**
- Updated CLI docs to refer renamed mbed_rtos

### CODE REVIEW

**Comment by Elise-Kaminski** [comment]
@rajkan01 
Hello, 
Am I the only reviewer for this change? Looks good to me. Let me know if I can merge (and port that on the 6.3 branch).
Thank you.

File: docs/tools/CLI/cli-test-debug.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Allocated Stack: 32768 bytes
 Total Static RAM memory (data + bss): 6568 bytes
 Total RAM memory (data + bss + heap + stack): 104876 bytes
 Total Flash memory (text + data + misc): 48357 bytes
-Image: build\tests\K64F\GCC_ARM\TESTS\mbedmicro-rtos-mbed\mutex\TESTS-unit-myclass.bin
+Image: build\tests\K64F\GCC_ARM\TESTS\mbed_rtos\mutex\TESTS-unit-myclass.bin
```
Comment (LDong-Arm) [inline docs/tools/CLI/cli-test-debug.md]: The actual path to `mbed_rtos` tests is `BUILD/tests/K64F/GCC_ARM/rtos/tests/TESTS/mbed_rtos/` after your PR in mbed-os.

But this line is probably just a random path the original author used for demo only, and the value doesn't really matter.

**Comment by evedon** [comment]
> @rajkan01
> Hello,
> Am I the only reviewer for this change? Looks good to me. Let me know if I can merge (and port that on the 6.3 branch).
> Thank you.

The associated PR is not in 6.3 so this will be for the next release.

**Comment by Elise-Kaminski** [comment]
> > @rajkan01
> > Hello,
> > Am I the only reviewer for this change? Looks good to me. Let me know if I can merge (and port that on the 6.3 branch).
> > Thank you.
> 
> The associated PR is not in 6.3 so this will be for the next release.

Hello,
Can it wait until Irit comes back then? I won't have time to review the entire file this week. If it's just a path change, I can approve and merge the change. Let me know.
Thank you.

File: docs/tools/CLI/cli-test-debug.md (hunk 1 above)
Comment (rajkan01) [reply docs/tools/CLI/cli-test-debug.md]: Yes, I updated with the correct path. Even though text is used for demo but still provide the right information would be better.

**Review by LDong-Arm — APPROVED** [review]

**Comment by rajkan01** [comment]
> 
> 
> > > @rajkan01
> > > Hello,
> > > Am I the only reviewer for this change? Looks good to me. Let me know if I can merge (and port that on the 6.3 branch).
> > > Thank you.
> > 
> > 
> > The associated PR is not in 6.3 so this will be for the next release.
> 
> Hello,
> Can it wait until Irit comes back then? I won't have time to review the entire file this week. If it's just a path change, I can approve and merge the change. Let me know.
> Thank you.

Hi @Elise-Kaminski , Yes, it is just path reference change. It would be good to merge if you are happy with the changes.

**Comment by Elise-Kaminski** [comment]
OK. Looks good to me.

**Pull request merged**